node/parachain/src/service.rs

```diff
@@ -45,6 +66,7 @@ type FullBackend = TFullBackend<Block>;
 #[allow(clippy::type_complexity)]
 pub fn new_partial(
 	config: &Configuration,
+	// author: H160,
```

Comment:
```suggestion
	author: H160,
```
@JoshOrndorff Before this is merged, we'll add back this input and remove the hard coded author below right?

Reply:
Yes, that was my intention. When I commented this and added the hardcoded one below I was thinking:

> I just want to make sure my inherent is even getting to the runtime. I won't waste time wiring this all the way through to the CLI before I even know the general approach is working.